core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/Partition.json

```diff
@@ -120,11 +128,13 @@
         },
         "JdbcSystemResourceOverride": {
             "wlst_type": "${JdbcSystemResourceOverride:JDBCSystemResourceOverrides}",
+            "online_bean": "unknown",
```

Comment:
@ddsharpe can we decide if we want to explicitly set the value to `unknown` or omit the attribute to indicate that the value is unknown?  It seems like omitting it is more consistent with the approach we take in the aliases so I think it is worth a quick discussion.

Reply:
Derek agrees with you. So by popular demand:
- replaced a few of the unknowns with their bean after a bit of research
- removed the remaining unknown unknowns

Fixed